src/riscv/lib/src/state_backend.rs

```diff
@@ -77,7 +77,7 @@ pub mod hash;
 mod layout;
 pub mod owned_backend;
 pub mod proof_backend;
-mod proof_layout;
+pub mod proof_layout;
```

Comment:
Can it be `pub(crate)`?

Reply:
Changed [here](https://github.com/tezos/riscv-pvm/compare/b21eeac2df94938e737c261a72a9f1622324f421..d0bf1489520137e2e5cbdfbe28f4ad5f890438b7)